packages/contracts/docs/osx/02-how-to-guides/02-plugin-development/07-publication/01-versioning.md

```diff
@@ -1,10 +1,15 @@
 ---
-title: Versioning
+title: Adding a new version of your plugin
 ---
 
-## Versioning Your Plugin
+## How to add a new version of your plugin
 
-The Aragon OSx protocol has an on-chain versioning system built-in and distinguishes between releases and builds.
+The Aragon OSx protocol has an on-chain versioning system built-in, which distinguishes between releases and builds.
+
+- **Releases** contain breaking changes, which are incompatible with preexisting installations. Updates to a different release are not possible. Instead, you must install the new plugin release and uninstall the old one.
```

Comment:
The previous sections (4-upgradeable-plugin) has been talking about builds without releases/builds being introduced formally